src/api/app/models/review.rb

```diff
@@ -302,6 +302,12 @@ def staging_project?
     for_project? && !project&.staging_workflow_id.nil?
   end
 
+  def check_reviewer!
+    return unless (selected_errors = errors.select { |error| error.attribute.in?([:user, :group, :project, :package]) }).any?
+
+    raise ::NotFoundError, selected_errors.map { |error| "#{error.attribute.capitalize} not found" }.to_sentence
```

Comment:
Do we often use this style?
Normally, I would expect `if errors => raise` with the non-error case as fallthrough.

Reply:
Changing the code to:
```ruby
  def check_reviewer!
    if (selected_errors = errors.select { |error| error.attribute.in?([:user, :group, :project, :package]) }).any?
      raise ::NotFoundError, selected_errors.map { |error| "#{error.attribute.capitalize} not found" }.to_sentence
    end
  end
```

... makes RuboCop complain:
```ruby
app/models/review.rb:306:5: C: [Correctable] Style/GuardClause: Use a guard clause (return unless (selected_errors = errors.select { |error| error.attribute.in?([:user, :group, :project, :package]) }).any?) instead of wrapping the code inside a conditional expression.
    if (selected_errors = errors.select { |error| error.attribute.in?([:user, :group, :project, :package]) }).any?
    ^^
```

Next time you could enter `bash` in the frontend container of the development environment (with `docker-compose run --rm --service-ports frontend bash`) and run `bin/rubocop --ignore_parent_exclusion app/models/review.rb`.

For reference see: https://github.com/openSUSE/open-build-service/wiki/Rubocop#running-rubocop-in-your-own

Reply:
looking at https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause , reformulating it as
`raise if errors`
should also be accepted.

Reply:
Changed. Please take a look again.